ui/window.go

```diff
@@ -254,28 +250,11 @@ func (win *Window) Loop(shutdown chan int) {
 				go func() {
 					win.keyEvents <- &evt
 				}()
-			case clipboard.Event:
-				go func() {
-					win.theme.Clipboard <- evt.Text
-				}()
 			case nil:
 				// Ignore
 			default:
 				log.Tracef("Unhandled window event %+v\n", e)
 			}
-		case e := <-win.clipboard:
-			switch c := e.(type) {
-			case decredmaterial.ReadClipboard:
-				win.window.ReadClipboard()
-			case WriteClipboard:
-				go func() {
-					win.window.WriteClipboard(c.Text)
-					win.toast <- &toast{
-						text:    "copied",
-						success: true,
-					}
-				}()
```

Comment:
As a result of the change, the toast displayed after triggering a copy event doesn't display anymore. I'll suggest using a global list for the toast, similar to how the modal is going to be refactored as described in issue #424.